lte/gateway/python/magma/mobilityd/BUILD.bazel

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+load("@subpar//:subpar.bzl", "par_binary")
+load("@python_deps//:requirements.bzl", "requirement")
+
+DEBS = [
+    ":ip_address_man",
+    "//lte/protos:keyval_python_proto",
+    "//lte/protos:mconfigs_python_proto",
+    "//lte/protos:mobilityd_python_grpc",
+    "//lte/protos:mobilityd_python_proto",
+    "//lte/protos:subscriberdb_python_grpc",
+    "//lte/protos:subscriberdb_python_proto",
+    "//orc8r/gateway/python/magma/common:metrics",
+    "//orc8r/gateway/python/magma/common:sentry",
+    "//orc8r/gateway/python/magma/common/redis:client",
+    "//orc8r/gateway/python/magma/configuration:service_configs",
+    "//orc8r/protos:eventd_python_grpc",
+    "//orc8r/protos:mconfig_python_proto",
+    "//orc8r/protos:mconfigs_python_proto",
+    "//orc8r/protos:metricsd_python_proto",
+    "//orc8r/protos:redis_python_proto",
+    "//orc8r/protos:service303_python_grpc",
+    "//orc8r/protos:service303_python_proto",
+]
+
+par_binary(
+    name = "mobilityd_par",
+    srcs = glob(["*.py"]),
+    # legacy_creat_init = False is required to fix issues in module import, see https://github.com/rules-proto-grpc/rules_proto_grpc/issues/145
+    legacy_create_init = False,
+    main = "main.py",
+    python_version = "PY3",
+    visibility = ["//visibility:public"],
```

Comment:
Ditto comments earlier on `py_test` targets, I don't think `par_binary` having public visibility (or any visibility specification) makes sense (correct me if I'm wrong / missing something!).

Reply:
Is changed.